public/stylesheets/site/2.0/07-interactions.css

```diff
@@ -21,15 +21,15 @@ fieldset {
   min-width: 0;
 }
 
-/*Guideline: Forms are written by lots of people, and, this is beta, change a LOT. 
-I've written a load of possible different nests to give a reasonably/broadly consistent view whatever goes in, 
+/*Guideline: Forms are written by lots of people, and, this is beta, change a LOT.
+I've written a load of possible different nests to give a reasonably/broadly consistent view whatever goes in,
 but it's probably not complete. */
 
 fieldset, form dl, fieldset dl dl, fieldset fieldset fieldset, fieldset fieldset dl dl, dd.hideme, form blockquote.userstuff {
   display: block;
   background: #ddd;
   border: 2px solid #f3efec;
-  margin: 0.643em;
+  margin: 0.643em auto;
```

Comment:
I think you're right. Sorry for not catching it when the issue was created! I'm going to close the issue and this pull request, but thank you for taking a shot at it.